releasenotes/notes/switched-to-StandardScaler-43d24a7918e96c14.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-upgrade:
+fixes:
   - |
-    The :meth:`.NaturalGradient._ridge` and :meth:`.NaturalGradient._lasso` methods
-    now use the :class:`~sklearn.preprocessing.StandardScaler` to scale the data
+    Fix deprecation warnings in :class:`.NaturalGradient`, which now uses the
+    now use :class:`~sklearn.preprocessing.StandardScaler` to scale the data
```

Comment:
Thank you, @Cryoris. I believe there's a repetition in the reno here - `now uses the` , `now use`.

Reply:
Oh yep, thank you!